.github/workflows/main.yml

```diff
@@ -42,6 +42,9 @@ jobs:
           python -m pip install --upgrade pip setuptools
           python -m pip install -r blackbox/requirements.txt
 
+          # Let's use Sphinx 3.x here because its link checker is much more capable.
```

Comment:
```suggestion
          # We default to sphinx 1.8 in blackbox/requirements.txt because that's what the crate-docs-theme needs. We want sphinx 3.x here for link checker improvements.
```